### DISCUSSION TITLE
Markets of event  is empty

### DISCUSSION THREAD

**Discussion by dhoaison** [cloudbet/docs#35] [Q&A] [answered]
Hi. I try to load  all available main markets by default but the result contains empty markets  

![image](https://user-images.githubusercontent.com/32816961/179006193-b92e20eb-c126-486c-8a3b-fe90dfeae3c4.png)

**Comment by kgravenreuth** [comment, marked as answer]
Hi @dhoaison 

Apologies for the late reply. For live events (i.e. event with status `TRADING_LIVE`) you may encounter periods where the `markets` are empty, since we may suspend live markets from time to time for risk management purposes. They markets usually re-open whenever we are offering them.

You can find a list of all our offered markets here as mentioned in this repo's Readme: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7
